.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+<!--
```

Comment:
is this comment appears up in template? view markdown in PR shows only non commented parts.